fluidsynth.py

```diff
@@ -5,7 +5,7 @@
 
     Python bindings for FluidSynth
 
-    Copyright 2008, Nathan Whitehead <nwhitehe@gmail.com>
+    Copyright 2008-2019, Nathan Whitehead <nwhitehe@gmail.com> and others
```

Comment:
2021. I know I'm a bit late to the party, apologies

Reply:
Yeah, sorry, I'm not going to be working on this PR any more. Too late for that. Feel free whatever you can from it or close it as you please.